### DISCUSSION TITLE
LogicFlow在vue2的应用中，内存泄露

### DISCUSSION THREAD

**Discussion by weijian-zhu** [didi/LogicFlow#1332] [Q&A] [answered]
LogicFlow在vue2的应用中，来回切换路由渲染流程图时，内存积累，不能释放，想问下LogicFlow实例会自动销毁嘛，我在文档中没找到主动销毁LogicFlow实例的方法，请问我该如何主动销毁，目前我在beforeDestroy钩子中 lf = null 的方式处理的

**Comment by wumail** [comment, marked as answer]
没有主动销毁实例的办法，lf = null是合理的
